docs/ui/auth/fragments/web/authenticator.md

```diff
@@ -83,6 +83,36 @@ _App.vue_
 
 </docs-filter>
 
+<amplify-callout warning>
+
+To enable password manager, you need to wrap the Authenticator component with `amplify-auth-container` component.
+
+<docs-filter framework="react">
+
+<inline-fragment src="~/ui/auth/fragments/react/password-manager.md"></inline-fragment>
+
+</docs-filter>
+
+<docs-filter framework="angular">
+
+<inline-fragment src="~/ui/auth/fragments/angular/password-manager.md"></inline-fragment>
```

Comment:
Seeing `/angular/` on this, Ionic and Vue always worries me a little 😅  Maybe we consider `/non-react/` in the future for the Next.js rewrite?

Reply:
Yeah, file naming conventions can be clearer. I like your idea -- just like how we have `/web/` to encompass the web frameworks. Maybe `jsx` vs `non-jsx`? `jsx` vs `html`? Naming things are hard...

Reply:
Yeah, it's definitely the hardest problem of programming 😆

Reply:
It's just on my mind recently because I've seen `/native-common/` a lot for encompassing fragments that are in the iOS/Android/Flutter/RN bucket!